packages/create-astro/src/messages.ts

```diff
@@ -1,126 +1,183 @@
-	`Let's claim your corner of the internet.`,
-	`I'll be your assistant today.`,
-	`Let's build something awesome!`,
-	`Let's build something great!`,
-	`Let's build something fast!`,
-	`Let's make the web weird!`,
-	`Let's make the web a better place!`,
-	`Let's create a new project!`,
-	`Let's create something unique!`,
-	`Time to build a new website.`,
-	`Time to build a faster website.`,
-	`Time to build a sweet new website.`,
-	`We're glad to have you on board.`,
-	`Keeping the internet weird since 2021.`,
-	`Initiating launch sequence...`,
-	`Initiating launch sequence... right... now!`,
-	`Awaiting further instructions.`,
-];
-
-export function getName() {
-	return new Promise((resolve) => {
-		exec('git config user.name', { encoding: 'utf-8' }, (_1, gitName, _2) => {
-			if (gitName.trim()) {
-				return resolve(gitName.split(' ')[0].trim());
-			}
-			exec('whoami', { encoding: 'utf-8' }, (_3, whoami, _4) => {
-				if (whoami.trim()) {
-					return resolve(whoami.split(' ')[0].trim());
-				}
-				return resolve('astronaut');
-			});
-		});
-	});
+let stdout = process.stdout;
+/** @internal Used to mock `process.stdout.write` for testing purposes */
+export function setStdout(writable: typeof process.stdout) {
+	stdout = writable;
 }
 
-let v: string;
-export function getVersion() {
-	return new Promise<string>((resolve) => {
-		if (v) return resolve(v);
-		get('https://registry.npmjs.org/astro/latest', (res) => {
-			let body = '';
-			res.on('data', (chunk) => (body += chunk));
-			res.on('end', () => {
-				const { version } = JSON.parse(body);
-				v = version;
-				resolve(version);
-			});
-		});
-	});
+export async function say(messages: string|string[], { clear = false, hat = '' } = {}) {
+	return houston(messages, { clear, hat, stdout });
 }
 
-export async function banner(version: string) {
-	return console.log(
-		`\n${label('astro', color.bgGreen, color.black)}  ${color.green(
-			color.bold(`v${version}`)
-		)} ${color.bold('Launch sequence initiated.')}\n`
-	);
+export async function spinner(args: { start: string; end: string; while: (...args: any) => Promise<any>; }) {
+	await load(args, { stdout });
 }
 
-export async function info(prefix: string, text: string) {
-	await sleep(100);
-	if (process.stdout.columns < 80) {
-		console.log(`${color.cyan('◼')}  ${color.cyan(prefix)}`);
-		console.log(`${' '.repeat(3)}${color.dim(text)}\n`);
-	} else {
-		console.log(`${color.cyan('◼')}  ${color.cyan(prefix)} ${color.dim(text)}\n`);
-	}
+export const title = (text: string) => align(label(text), 'end', 7) + ' ';
+
+export const welcome = [
+    `Let's claim your corner of the internet.`,
+    `I'll be your assistant today.`,
+    `Let's build something awesome!`,
+    `Let's build something great!`,
+    `Let's build something fast!`,
+		`Let's build the web we want.`,
+    `Let's make the web weird!`,
+    `Let's make the web a better place!`,
+    `Let's create a new project!`,
+    `Let's create something unqiue!`,
```

Comment:
```suggestion
    `Let's create something unique!`,
```
I mean, I almost didn't want to change it!  :joy:

Reply:
Haha, incredible!